.github/workflows/pr.yml

```diff
@@ -112,6 +112,6 @@ jobs:
         fi
     
     - name: Archive Debug Build
-      uses: kittaakos/upload-artifact-as-is@v0
+      uses: actions/upload-artifact@v3
```

Comment:
The PR flow now includes one file - artifact, 82 MB - instead of all APKs

Reply:
It's less convenient, but not a major problem. Just wanted to point it out :)

Reply:
That action was archived and throwing warnings so figured it was safer to use a more default one.